### PR TITLE
add loader for delete profile and update profile status

### PR DESCRIPTION
**Pull request by purisaurabh**
This PR includes the loader for the delete profile and the updating profile status

### CODE REVIEW

File: src/api/store/store.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import { GetLocalStorage } from "../../helpers";
```
Comment (Payal13) [inline src/api/store/store.jsx]: function name casing is wrong - getLocalStorage, setLocalStorage

File: src/components/Profile/List/index.jsx [hunk 2]
```diff
@@ -374,7 +376,11 @@ const ListProfiles = () => {
 
   return (
     <>
-      <Spin spinning={isLoading} size={SPIN_SIZE} tip={"Just a moment.."}>
+      <Spin
+        spinning={isLoading || profileStatusUpdating || deletingProfile}
+        size={SPIN_SIZE}
+        tip={"Just a moment.."}
```
Comment (Payal13) [inline src/components/Profile/List/index.jsx]: use loading constant here

File: src/components/Resume/index.js [hunk 3]
```diff
@@ -368,13 +375,17 @@ const Resume = forwardRef(({ data }, ref) => {
...
               profile_id: profile?.id,
             });
-            await logoutService();
-            dispatch(logout());
-            window.localStorage.clear();
 
             if (response?.data) {
               toast.success(response?.data?.message, SUCCESS_TOASTER);
-              navigate(ROOT_ROUTE);
+              if (role.toLowerCase() === "admin") {
```
Comment (Payal13) [inline src/components/Resume/index.js]: use role name constants

File: src/api/store/store.jsx (hunk 1 above)
Comment (purisaurabh) [reply src/api/store/store.jsx]: done

File: src/components/Profile/List/index.jsx (hunk 2 above)
Comment (purisaurabh) [reply src/components/Profile/List/index.jsx]: updated

File: src/components/Resume/index.js (hunk 3 above)
Comment (purisaurabh) [reply src/components/Resume/index.js]: updated

**Review by Payal13 — APPROVED** [review]

**Review by Payal13 — APPROVED** [review]

**Pull request merged**